catalog-info.yaml

```diff
@@ -37,6 +37,9 @@ spec:
     spec:
       repository: elastic/fleet-server
       pipeline_file: ".buildkite/pipeline.yml"
+      provider_settings:
+        build_pull_request_forks: false
+        build_pull_requests: false # PR builds are managed by buildkite-pr-bot

```

Comment:
the buildkite-pr-bot will be added shortly so the `are` for now is ok to be used in this context for now.